.mailmap

```diff
@@ -1700,6 +1700,7 @@ carstimon <carstimon@gmail.com>
 ck Lux <lux.r.ck@gmail.com>
 cocolato <haiizhu@outlook.com> Hai Zhu <35182391+cocolato@users.noreply.github.com>
 codecruisader <nnisarg55@gmail.com>
+cp50 <chrispaul68002@gmail.com>
```

Comment:
This should usually be something like:
```
Chris Paul <chrispaul68002@gmail.com> cp50 <chrispaul68002@gmail.com>
```
The first part is the name that will be added (later) to the AUTHORS file.